### PR TITLE
Add bind::zone forwarders parameter

### PR DESCRIPTION
**Pull request by jeffmccune**
Without this patch a zone cannot be configured with a forward zone. This patch
adds the zone forwarders parameter, modeled on the allow-updates parameter.

### CODE REVIEW

File: manifests/zone.pp [hunk 1]
```diff
@@ -33,6 +33,7 @@
   $type            = undef, # master or slave
   $update_policies = undef, # mutually exclusive with allow_update
   $allow_update    = undef,
+  $forwarders      = undef,
```
Comment (ghoneycutt) [inline manifests/zone.pp]: Please document this in the README

File: templates/zone.erb [hunk 2]
```diff
@@ -19,6 +19,18 @@ zone "<%= @zone %>" {
   allow-update { <%= au.join('; ') %>; };
 <% end -%>
   file "<%= @dir %><% if not @extra_path.nil? -%><%= @extra_path %><% end -%>/<%= @zone %>";
+<% if @forwarders -%>
+<%   fwlst = Array.new -%>
+<%   @forwarders.each do |i| -%>
+<%     if i =~ /^key/ -%>
+<%       # add quotes around name of key -%>
+<%       fwlst << "key \"#{i.split[1]}\"" -%>
```
Comment (ghoneycutt) [inline templates/zone.erb]: Good catch!

File: manifests/zone.pp (hunk 1 above)
Comment (jeffmccune) [reply manifests/zone.pp]: Documented

File: templates/zone.erb (hunk 2 above)
Comment (jeffmccune) [reply templates/zone.erb]: Thanks, copied it from the allow_updates behavior.

**Comment by jeffmccune** [comment]
@ghoneycutt ready to rock.

**Comment by ghoneycutt** [comment]
Thank you!

Released in v2.2.0

**Pull request merged**